bindings/pyroot/pythonizations/python/ROOT/_pythonization/_rntuple.py

```diff
@@ -60,6 +60,12 @@ def _RNTupleModel_CreateBare(*args):
     return ROOT.Experimental.RNTupleModel._CreateBare()
 
 
+def _RNTupleModel_CreateEntry(self):
+    raise RuntimeError(
+        "creating entries from model is not supported in Python, create CreateEntry on the reader or writer"
```

Comment:
```suggestion
        "creating entries from model is not supported in Python, call CreateEntry on the reader or writer"
```

Reply:
Ew, that happens if I rewrite the error message too many times...